### DISCUSSION TITLE
runplan fail

### DISCUSSION THREAD

**Discussion by syitzhm** [frePPLe/frepple#422] [Q&A]
Hello:
 
    I am trying to bring up frepple engine on my environment, and everything seems works fine, but when I execute "launch the engine", I met issue that "Failed with exit code 1", and I found in the code there is a line "import frepple", and other lines refer to this module are wrong, and I did not find this class or module everywhere, would you please guide me how to fix this? Thanks.

**Comment by jdetaeye** [comment]
Python files where you see "import frepple" are expected to be executed by the frepple executable (that will run it in its embedded python interpreter).  So, by looking for a python module called frepple you're on the wrong path.

To find the real root cause of the "failed with exit code 1", try running the executable "frepple" from the command line. 
Possible causes are a) missing shared libraries of some sort, b) missing or invalid license file in the enterprise edition

**Reply by syitzhm** [reply]
Hello:
    Thanks for quick reply, I did not find this executable file everywhere under the whole directory, how could I execute this file? Sorry this may be a stupid question, please help. BTW, I am under the Mac OS environment.
